lang/en/local_csp.php

```diff
@@ -45,7 +45,9 @@
 $string['cspsettings'] = 'Content security policy settings';
 $string['cspsettingsinfo'] = '<p>CSP works through adding a special HTTP response header to every Moodle page. Modern browsers, when they see this header, are able to perform certain actions e.g. block insecure content on such pages. Please read more about CSP <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP">here</a>.</p><p>If you leave any of these settings blank CSP headers will not be used.</p>';
 $string['documenturi'] = 'Violation at';
-$string['failcounter'] = 'Count';
+$string['failcounter'] = '#';
+$string['highestviolaters'] = 'Top Violation Sources';
+$string['highestviolaterscount'] = 'Count: {$a}';
```

Comment:
didn't want to simplify this to just?:

10 blah
9 foo
8 err

Reply:
This looked a little bit cleaner, with just the number it looked a little unfinished